Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.py

```diff
@@ -67,12 +72,13 @@ def __init__(self, client_id: str, organization: str, verify: bool, proxy: bool,
         client_args = assign_params(
             self_deployed=True,
             auth_id=client_id,
-            token_retrieval_url='https://login.microsoftonline.com/organizations/oauth2/v2.0/token',
+            token_retrieval_url='https://login.microsoftonline.com/organizations/oauth2/v2.0/token' if 'Device Code' in
+                                                                                                       auth_type else None,
             grant_type=GRANT_BY_CONNECTION[auth_type],
             base_url=f'https://dev.azure.com/{organization}',
             verify=verify,
             proxy=proxy,
-            scope=AZURE_DEVOPS_SCOPE,
+            scope=SCOPE_CLIENT_CREDENTIALS_FLOW if 'Client Credentials' in auth_type else SCOPE_DEVICE_AUTH_FLOW,
```

Comment:
I think the new scope is also relevant for 'Authorization Code' flow, so maybe change it to if not ''Device Code' (we can check it)